vllm/worker/habana_model_runner.py

```diff
@@ -942,10 +956,16 @@ def log_warmup(self, phase, i, max_i, batch_size, seq_len):
         logger.info(f"[Warmup][{phase}][{i+1}/{max_i}] batch_size:{batch_size} seq_len:{seq_len} free_mem:{free_mem}")
 
     def warmup_all_buckets(self, buckets, is_prompt, kv_caches):
+        counter = 0
         for i, (batch_size, seq_len) in enumerate(reversed(buckets)):
             mem_usage = 100.0 * HabanaMemoryProfiler.current_device_memory_usage() / HabanaMemoryProfiler.total_device_memory()
             self.log_warmup('Prompt' if is_prompt else 'Decode', i, len(buckets), batch_size, seq_len)
-            self.warmup_scenario(batch_size, seq_len, is_prompt, kv_caches)
+            try:
+                self.warmup_scenario(batch_size, seq_len, is_prompt, kv_caches)
+            except:
+                print(f"Failed on scenario {i+1}: batch_size={batch_size}, seq_len={seq_len}, is_prompt={is_prompt}")
+                counter += 1
+        print(f"Failed warm-up scenarios = {counter}")
```

Comment:
warmup should not fail, and if it does, we should not suppress those failures

Reply:
Removed